contrib/BBEditTextWrangler-julia.plist

```diff
@@ -238,7 +238,7 @@
             <string>broadcast_setindex!</string>
             <string>bswap</string>
             <string>bytes2hex</string>
-            <string>bytestring</string>
+            <string>String</string>
```

Comment:
already present in a separate list of types later in the file. if it weren't, would be nice to keep this file sorted